web/src/pages/api/v1/precheck/[app_id].ts

```diff
@@ -189,7 +204,25 @@ export default async function handlePrecheck(
       req
     );
   }
-
+  const app_metadata =
+    rawAppValues.app_metadata.length > 0
```

Comment:
is this check even needed, if length is 0, then `rawAppValues.app_metadata[0]` will be `undefined` anyways